### DISCUSSION TITLE
I cant doawnload Revanced on Bleustack emulator

### DISCUSSION THREAD

**Discussion by Tyrozz34** [ReVanced/revanced-discussions#886] [Question]
Hello ! i have some troubles to instal revanced manager, and revanced Youtube on the BleuStack PC android emulator, it say that he can do an "analyse" on the APK when i opent -it, like instantly ! 

if somebody know why, thanks ! 

*i used Youtube vanced before and any trouble with-ti except for today*

**Comment by oSumAtrIX** [comment]
Please reformulate your message, it is incomprehensible

**Comment by sahilkumarnishant** [comment]
> Hello ! i have some troubles to instal revanced manager, and revanced Youtube on the BleuStack PC android emulator, it say that he can do an "analyse" on the APK when i opent -it, like instantly ! 
> 
> if somebody know why, thanks ! 
> 
> *i used Youtube vanced before and any trouble with-ti except for today* 

I don't have a solution for this . But if you want to use ad free YouTube in pc then use YouTube in Brave browser :)

**Comment by REVENGE977** [comment]
why would you want revanced on your pc tho, just install these extensions on your browser:
- ublock origin
- Enhancer for Youtube
- SponserBlock
- Return Youtube Dislikes 

and you will get the same features.
